### ISSUE TITLE
APP CaixaBank-Now refuses to make operations

### ISSUE THREAD

**Issue by Nacheras** [snake-4/Zygisk-Assistant#79]
**Describe the bug**

The app needs a signature activation to work, bit it refuses to activate it, so I´m not able to add cards to GPay, make bizums, transactions..... I can do nothing
Here is the app [link](https://play.google.com/store/apps/details?id=es.lacaixa.mobile.android.newwapicon&pcampaignid=web_share)

**Steps To Reproduce**

1. Once I log in, I enter in Profile -> Security -> Signature -> Activate Signature
2. Then I get an error and I can´t proceed to activate the Signature, I think it´s caused because it still detects root and unloocked bootloader

**Context**

 - Device: Xioami Mi12 lite 5G (taoyao)
 - OS: CrDroid 11
 - Version of Magisk, KSU or APatch: Magisk 28.1
 - Other Root Module(s): Play Integrity Fix
 - LSPosed Module(s): None

**Logcat**
 [log.txt](https://github.com/user-attachments/files/18269547/log.txt)

**Comment by JimZam** [comment]
Good evening. I don't know if you already solved it, I hope so. If bot maybe I can help you. Write me a privare message.
Un saludo

**Comment by Nacheras** [comment]
Hi! Do you speak spanish??
I couldn´t find a solution and caixabank app refused to work, I had to relock bootlader and reflash the original ROM, a pitty.
I´m interested in getting it to work if you can help me

**Comment by JimZam** [comment]
Yes, I do. Do you use Telegram? Look for "Magisk Grupo Español" and post there your question. I'll try to help you.

**Comment by Nacheras** [comment]
Hay muchos grupos de español de magisk en google, tienes el link?

**Comment by JimZam** [comment]
https://t.me/MagiskEspanol

**Comment by Nacheras** [comment]
Gracias!

**Comment by snake-4** [comment]
Can you try with ZygiskNext instead of stock Zygisk?

**Comment by Nacheras** [comment]
I´ll try and tell the results soon

**Comment by CarlosEkisde** [comment]
I confirm it's working on CrDroid 11.5 (Android 15) with ZygiskNext

**Comment by Nacheras** [comment]
> I confirm it's working on CrDroid 11.5 (Android 15) with ZygiskNext

Pero te ha dejado activar la firma en el móvil?? Estoy de la Caixa hasta más arriba

**Comment by CarlosEkisde** [comment]
> > I confirm it's working on CrDroid 11.5 (Android 15) with ZygiskNext
> 
> Pero te ha dejado activar la firma en el móvil?? Estoy de la Caixa hasta más arriba

Sí, he logrado que funcionase de una puta vez XD. Funciona tanto con ZygiskNext como con ReZygisk.

**Comment by Nacheras** [comment]
> Sí, he logrado que funcionase de una puta vez XD. Funciona tanto con ZygiskNext como con ReZygisk.

Probaré , a ver si tengo suerte

**Comment by Nacheras** [comment]
> > > I confirm it's working on CrDroid 11.5 (Android 15) with ZygiskNext
> > 
> > 
> > Pero te ha dejado activar la firma en el móvil?? Estoy de la Caixa hasta más arriba
> 
> Sí, he logrado que funcionase de una puta vez XD. Funciona tanto con ZygiskNext como con ReZygisk.

A funcionado!!!!
Menos mal, ahora a ver como añadir la tarjeta al Wallet que estos de Google se han puesto muy puñeteros también

**Comment by asterlxx** [comment]
tengo el mismo problema para activar caixabanknow tengo la version 5.83
uso los siguientes modulo
playintegrity fik inject
shamiko
tickstore
tsupport advance
zygist assistant
zygist next
zygist detach

bank app is in deny list

es la unica que no funciona.. alguna recomendacion??

**Comment by Nacheras** [comment]
> tengo el mismo problema para activar caixabanknow tengo la version 5.83 uso los siguientes modulo playintegrity fik inject shamiko tickstore tsupport advance zygist assistant zygist next zygist detach
> 
> bank app is in deny list
> 
> es la unica que no funciona.. alguna recomendacion??

Sigue [estos ](https://t.me/popMODS/6548)pasos al pie de la letra y ya verás como funciona la caixa con huella digital y todo

**Comment by jmsegarra** [comment]
> > > I confirm it's working on CrDroid 11.5 (Android 15) with ZygiskNext
> > 
> > 
> > Pero te ha dejado activar la firma en el móvil?? Estoy de la Caixa hasta más arriba
> 
> Sí, he logrado que funcionase de una puta vez XD. Funciona tanto con ZygiskNext como con ReZygisk.

Como has conseguido que funcione? Tengo Los 22.2 y no puedo activar la firma en el dispositivo, cuando lo intento se quedan los tres puntitos saltando eternamente. He probado con los pasos de Nacheras y tampoco

**Comment by CarlosEkisde** [comment]
¿Alguno vuelve a tener problemas desde la versión 5.85? De momento estoy sobreviviendo gastando una versión anterior que sí funciona.

**Comment by Nacheras** [comment]
> ¿Alguno vuelve a tener problemas desde la versión 5.85? De momento estoy sobreviviendo gastando una versión anterior que sí funciona.

Yo estoy desesperado con esto de la integridad, formateé el móvil la semana pasada para actualizar a Android 16, instalo todo de nuevo y meto los módulos necesarios para obtener los 3 ticks de integridad, todo de maravilla hasta pasadas unas 2h desde que pude configurar el banco sin problemas y me dice que ya no paso la integridad, me salen 3 cruces rojas, y no he tocado nada!!
No se como hacen para detectarlo estos de Google, que más les dará que el móvil esté liberado!!

**Comment by asterlxx** [comment]
Hola 
nose si usaran TWRP para cargar una rom o para instalar magisk o para actualizar o para cargar algun plugin.... pero descubir que TWRP pone una carpeta TWRP dentro de la carpeta SDCARD... borrandola mas teniendo tricky store+zygisk-detach+play integrity fix inject, funciona bien caixa.

**Comment by CarlosEkisde** [comment]
> > ¿Alguno vuelve a tener problemas desde la versión 5.85? De momento estoy sobreviviendo gastando una versión anterior que sí funciona.
> 
> Yo estoy desesperado con esto de la integridad, formateé el móvil la semana pasada para actualizar a Android 16, instalo todo de nuevo y meto los módulos necesarios para obtener los 3 ticks de integridad, todo de maravilla hasta pasadas unas 2h desde que pude configurar el banco sin problemas y me dice que ya no paso la integridad, me salen 3 cruces rojas, y no he tocado nada!! No se como hacen para detectarlo estos de Google, que más les dará que el móvil esté liberado!!

Pero creo que La Caixa nunca ha requerido la integridad completa. No sé exactamente cómo lo harán pero yo nunca la he pasado ni me he molestado mucho en pasarla porque nunca lo he necesitado. Ahora veremos con las versiones nuevas qué  carajos piden.

> Hola nose si usaran TWRP para cargar una rom o para instalar magisk o para actualizar o para cargar algun plugin.... pero descubir que TWRP pone una carpeta TWRP dentro de la carpeta SDCARD... borrandola mas teniendo tricky store+zygisk-detach+play integrity fix inject, funciona bien caixa.

Pues sí que la tengo, sí... Tendré que probar lo que dices. Aunque ayer entre pitos y flautas deshabilité la firma de la aplicación de mi teléfono y probé a pagar sin ella. Y descubrí que cuando no tienes firma para confirmar un pago te mandan un código por SMS. Pues creo que lo voy a dejar así, ni firma ni hostias. Hasta que no me obliguen no voy a volver a intentar meter la basura de la firma en mi teléfono XD.

Que por cierto los SMS son más inseguros de largo que una rom modificada que el usuario instala por su propia cuenta y riesgo... Pero bueno, es la lógica de estas empresas de mierda.

**Comment by Nacheras** [comment]
> Pues sí que la tengo, sí... Tendré que probar lo que dices. Aunque ayer entre pitos y flautas deshabilité la firma de la aplicación de mi teléfono y probé a pagar sin ella. Y descubrí que cuando no tienes firma para confirmar un pago te mandan un código por SMS. Pues creo que lo voy a dejar así, ni firma ni hostias. Hasta que no me obliguen no voy a volver a intentar meter la basura de la firma en mi teléfono XD.
> 
> Que por cierto los SMS son más inseguros de largo que una rom modificada que el usuario instala por su propia cuenta y riesgo... Pero bueno, es la lógica de estas empresas de mierda.

Depende del recovery que uses, yo por ejemplo tengo Orange FOX y no crea ninguna carpeta llamada TWRP en la raíz del almacenamiento.
De todas formas, desde hace un años aproximadamente te obligan a tener la firma para hacer cualquier operación, y en la aplicación de Imagin hasta hace poco también, ya no son válidos los SMS.

**Comment by CarlosEkisde** [comment]
> > Pues sí que la tengo, sí... Tendré que probar lo que dices. Aunque ayer entre pitos y flautas deshabilité la firma de la aplicación de mi teléfono y probé a pagar sin ella. Y descubrí que cuando no tienes firma para confirmar un pago te mandan un código por SMS. Pues creo que lo voy a dejar así, ni firma ni hostias. Hasta que no me obliguen no voy a volver a intentar meter la basura de la firma en mi teléfono XD.
> > Que por cierto los SMS son más inseguros de largo que una rom modificada que el usuario instala por su propia cuenta y riesgo... Pero bueno, es la lógica de estas empresas de mierda.
> 
> Depende del recovery que uses, yo por ejemplo tengo Orange FOX y no crea ninguna carpeta llamada TWRP en la raíz del almacenamiento. De todas formas, desde hace un años aproximadamente te obligan a tener la firma para hacer cualquier operación, y en la aplicación de Imagin hasta hace poco también, ya no son válidos los SMS.

Pues los habrán vuelto a poner porque acabo de pagar 1000€ por tarjeta confirmando por SMS. Lo único que he tenido que hacer es deshabilitar la firma en el dispositivo en el que estaba activa.

**Comment by thejavix** [comment]
No se cómo lo hacéis yo no soy capaz con un s24 ultra.. siempre me falla el tema de la firma me detecta que el dispositivo está modificado. Obviamente tiene root con sukisu y bootloader abierto.. puedo hacer todo lo demás .. pagar con Wallet y tengo los tres check en verde.. 

Tiene que haber otra cosa..

**Comment by daviddavo** [comment]
Mismo problema aquí con un Xiaomi Redmi Note 10 pro con PixelOS. He probado muchos módulos y nada, sigue dando error.

Aunque para ser sinceros no he conseguido que ninguna app a parte de "Integrity Checker" me funcione... Incluso Native Detector sabe que está instalado el Magisk.
